src/odb/include/odb/db.h

```diff
@@ -425,7 +420,7 @@ class dbDatabase : public dbObject
   ///
   /// Create an instance of a database
   ///
-  static dbDatabase* create();
+  static dbDatabase* create(utl::Logger* logger = nullptr);
```

Comment:
Same question from the previous PR:  why wouldn't we require a logger.  I don't see why the db should ever create a new one.